pages/roadmap.js

```diff
@@ -175,12 +174,286 @@ export default function RoadmapPage() {
               title="Warning for Contributors"
               description="This roadmap reflects the priorities of the AsyncAPI Initiative. If you want to contribute a feature from the roadmap, feel free to let us know on the corresponding GitHub issue so we can discuss what's the best way to proceed and implement it yourself."
             />
-            <p className="text-xs text-center text-gray-600 mt-8">
-              <strong>Attention:</strong> this road map is synchronized with the GitHub issues in the <a href="https://github.com/asyncapi/shape-up-process/issues?q=is%3Aopen+is%3Aissue+label%3AOutcome" target="_blank" rel="noopener noreferrer" className="underline hover:text-gray-900">asyncapi/shape-up-process</a> repository.
-            </p>
           </div>
         </div>
       </div>
     </GenericLayout>
   )
 }
+
+const solutions = {
```

Comment:
The data powering the roadmap is here. Not perfect but very handy :)

Reply:
I don't have a problem with this code, but if it is to be updated once every 3 months wouldn't it be better to do it in the form of a yaml file - you don't have to do it now but it's my suggestion to make it easier to update the content on the roadmap in the future. WDYT? :)

Reply:
I don't think so. Having it here lets me have JSX code in the descriptions, which is very helpful.